pyproject.toml

```diff
@@ -1,24 +1,28 @@
-[tool.poetry]
+[project]
 name = "zcash-developer-tools"
 version = "0.1.0"
-description = "Zcash Developer Tools"
+dependencies = [
+  "drest",
+  "networkx",
+  "pygraphviz",
+  "sgqlc",
+  "str2bool",
+]
+requires-python = ">= 3.8"
 authors = [
-    "Jack Grigg <jack@electriccoin.co>",
+  {name = "Jack Grigg", email = "jack@electriccoin.co>"},
```

Comment:
```suggestion
  {name = "Jack Grigg", email = "jack@electriccoin.co"},
```

Reply:
Huh weird, I did not have this issue at all locally.